### PR TITLE
Consistency

### PR DESCRIPTION
**Pull request by pascaldekloe**
Tackle hand full of minors.

### CODE REVIEW

File: serialization_benchmarks_test.go [hunk 1]
```diff
@@ -131,7 +130,7 @@ func benchUnmarshal(b *testing.B, s Serializer) {
 		o := &A{}
 		err := s.Unmarshal(ser[n], o)
 		if err != nil {
-			b.Fatalf("%s failed to unmarshal: %s (%s)", s, err, ser[n])
```
Comment (alecthomas) [inline serialization_benchmarks_test.go]: This change seems less helpful, what's the purpose?

Comment (alecthomas) [reply serialization_benchmarks_test.go]: Ah I see what your goal was, but b.Helper() does not achieve what the old code did.

Comment (pascaldekloe) [reply serialization_benchmarks_test.go]: I think the old code had this String description to trace back the respective benchmark. Was there another achievement?

Comment (alecthomas) [reply serialization_benchmarks_test.go]: Yes it makes it easier to debug when something fails.

Comment (pascaldekloe) [reply serialization_benchmarks_test.go]: Printing binaries as UTF seems unwise. It also seems unlikely you'd want to debug in a serialisation benchmark. It is important to report all errors, so one can at least see when things go wrong. Happy to put a hex dump with string escape ("%x (%q)") back if you want it?

Comment (alecthomas) [reply serialization_benchmarks_test.go]: Yes, please revert.

Comment (alecthomas) [reply serialization_benchmarks_test.go]: Thanks! Hex dump is a good idea 👍

**Comment by alecthomas** [comment]
Thanks!

**Pull request merged**